packages/iov-keycontrol/src/wallets/slip10wallet.ts

```diff
@@ -253,7 +270,16 @@ export class Slip10Wallet implements Wallet {
         throw new Error("Unknown curve");
     }
 
-    const newIdentity = this.buildIdentity(chainId, pubkeyBytes);
+    return Slip10Wallet.buildIdentity(this.curve, chainId, pubkeyBytes);
+  }
+
+  public async createIdentity(chainId: ChainId, options: unknown): Promise<PublicIdentity> {
+    if (!isPath(options)) {
+      throw new Error("Did not get the correct argument type. Expected array of Slip10RawIndex");
+    }
+    const path = options;
+
+    const newIdentity = await this.previewIdentity(chainId, options);
```

Comment:
Yes, good reuse. Preview then save.
Also ensures that preview and create always work the same